dict/prh-idiomatic-usage.yml

```
@@ -243,6 +243,13 @@ rules:
     specs:
       - from: zipファイル
         to: ZIPファイル
+  - expected: アイコン
+    pattern:
+      - マーク
+    prh: 「マーク」ではなく「アイコン」と表記する
```

#266 と同じく、ガイドラインへのリンクを追加すると良さそうです！
```suggestion
    prh:  >-
    「マーク」ではなく「アイコン」と表記する
    https://smarthr.design/products/contents/idiomatic-usage/usage/
```